packages/medusa/src/api-v2/store/carts/query-config.ts

```diff
@@ -42,6 +42,12 @@ export const defaultStoreCartFields = [
   "region.name",
   "region.currency_code",
   "sales_channel_id",
+
+  // TODO: To be updated when payment sessions are introduces in the Rest API
+  "payment_collection",
```

Comment:
suggestion: rm, first level relation are not needed if we specifiy relation plus columns

Reply:
Apparently this didn't doesn't work. My latest commit fails on an undefined payment collection in the cart response 🤔